security/security-admin-module/pom.xml

```diff
@@ -29,7 +29,7 @@
         <dependency>
             <groupId>ddf.admin.core</groupId>
             <artifactId>admin-core-appservice</artifactId>
-            <version>1.1.1</version>
+            <version>2.8.0-SNAPSHOT</version>
```

Comment:
Since this is in the same repo now. We should be able to use ${project.version} here. The platform-error-servlet dependency (right below this) shows an example of that.